launcher/mainwindow_moc.cpp

```diff
@@ -116,10 +116,42 @@ MainWindow::MainWindow(QWidget * parent)
 
 	ui->settingsView->setDisplayList();
 	
+	if(settings["launcher"]["preLauncher"].Bool()) // MessageBox before launcher asking how to start (automatically starts client after 10 seconds)
+		preLauncher();
+
 	if(settings["launcher"]["updateOnStartup"].Bool())
 		UpdateDialog::showUpdateDialog(false);
 }
 
+void MainWindow::preLauncher()
+{
+	QMessageBox msgBox;
+	int secondsRemaining = 10;
+	auto text = "How to start?\n\nVCMI starts automatically in %n second(s)";
+	msgBox.setText(tr(text, "", secondsRemaining));
```

Comment:
`tr` doesn't work with variables

```suggestion
	msgBox.setText(tr("How to start?\n\nVCMI starts automatically in %n second(s)", nullptr, secondsRemaining));
```

Reply:
Then adding `tr("How to start?\n\nVCMI starts automatically in %n second(s)", nullptr, secondsRemaining)` twice in source?

Reply:
better use lambda to create the string